src/widgets/restFullClient/model/requestHandler.ts

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { IRequestData } from './requestHandlerModel';
+
+export const sendRequest = async (DATA: IRequestData) => {
+  const { method, url, headers, body } = DATA;
+  try {
+    const response = await axios({
+      method,
+      url,
+      headers,
+      data: body,
```

Comment:
И если исправить пропсы то:
```suggestion
      data,
```

Reply:
исправил